server/catgenome/src/main/java/com/epam/catgenome/entity/index/VcfIndexEntry.java

```diff
@@ -52,6 +52,7 @@
     private String geneName; //TODO: remove, use geneNameList
     private String geneNames;
     private String failedFilter;
+    private String sampleName;
```

Comment:
As I mentioned above, we don't split variation by sample, so no need for single `sampleName` field